lib/index.js

```diff
@@ -224,7 +237,7 @@ internals.models = (serverFrom, realmPath) => {
             return collector.models;
         }
 
-        const knexGroup = Hoek.reach(this, `${realmPath}.plugins.schwifty.knexGroup`);
+        const knexGroup = getKnexGroup(this);
```

Comment:
The way `reacher()` gets the perf improvement is by being called once to generate a reusable function.  That one call is where all the "overhead" of `Hoek.reach()` is done upfront.  Then the reusable function (in this case `getKnexGroup()`) doesn't have to do it, and you get perf wins when it matters.  For example, I didn't replace the call to `Hoek.reach()` in the initialization routine, because it's not a hot code path.  If you ever see `Toys.reacher(chain)(obj)` then you're not getting the perf wins from toys.